test/e2e/split/main.go

```diff
@@ -103,14 +112,14 @@ func findDescribes(dir string) ([]string, error) {
 		}
 		specNames := topDescribeRE.FindAllSubmatch(b, -1)
 		if len(specNames) == 0 {
-			log.Printf("%s: found no top level describes, skipping", match)
+			logger.Warnf("%s: found no top level describes, skipping", match)
```

Comment:
Another implementation: we could also just remove logging from this package entirely. It's nice for debugging this package while developing locally, but provides little value when running in CI environments.

Reply:
If it's not getting in the way, I'd say leave it. Logging is helpful during development. Though a debugger is also nice...